src/plugins/intel_cpu/src/ngraph_transformations/convert_fq_rnn_to_quantized_rnn.cpp

```diff
@@ -0,0 +1,226 @@
+#include <ngraph_ops/type_relaxed.hpp>
+#include <stdexcept>
+#include <vector>
+
+ov::intel_cpu::ConvertFqRnnToQuantizedRnn::ConvertFqRnnToQuantizedRnn() {
+    MATCHER_SCOPE(ConvertFqRnnToQuantizedRnn);
+
+    auto X_m = ngraph::pattern::any_input();
+    auto convert_X = ngraph::pattern::wrap_type<ngraph::op::Convert>({X_m});
+    auto input_shift_X = ngraph::pattern::wrap_type<ngraph::op::Constant>();
+    auto subtract_X = ngraph::pattern::wrap_type<ngraph::op::v1::Subtract>({convert_X, input_shift_X});
+    auto input_scale_X = ngraph::pattern::wrap_type<ngraph::op::Constant>();
+
+    auto deq_X = std::make_shared<ngraph::pattern::op::Or>(
+        OutputVector{
+            ngraph::pattern::wrap_type<ngraph::op::v1::Multiply>({convert_X, input_scale_X}),
+            ngraph::pattern::wrap_type<ngraph::op::v1::Multiply>({subtract_X, input_scale_X}),
+        });
+
+    auto H_m = ngraph::pattern::any_input();
+    auto convert_H = ngraph::pattern::wrap_type<ngraph::op::Convert>({H_m});
+    auto input_shift_H = ngraph::pattern::wrap_type<ngraph::op::Constant>();
+    auto subtract_H = ngraph::pattern::wrap_type<ngraph::op::v1::Subtract>({convert_H, input_shift_H});
+    auto input_scale_H = ngraph::pattern::wrap_type<ngraph::op::Constant>();
+
+    auto deq_H = std::make_shared<ngraph::pattern::op::Or>(
+        OutputVector{
+            ngraph::pattern::wrap_type<ngraph::op::v1::Multiply>({convert_H, input_scale_H}),
+            ngraph::pattern::wrap_type<ngraph::op::v1::Multiply>({subtract_H, input_scale_H}),
+        });
+
+    auto H_as_const = ngraph::pattern::wrap_type<ngraph::op::Constant>();
+    auto H_in = std::make_shared<ngraph::pattern::op::Or>(
+        OutputVector {
+            deq_H,
+            H_as_const
+        });
+
+    auto cell_state_m = ngraph::pattern::any_input(); // for LSTM
+    auto sequence_length_m = ngraph::pattern::any_input(); // for Sequences
+
+    auto W_m = ngraph::pattern::wrap_type<ngraph::op::Constant>();
+    auto convert_W = ngraph::pattern::wrap_type<ngraph::op::Convert>({W_m});
+    auto weights_scale_W = ngraph::pattern::wrap_type<ngraph::op::Constant>();
+    auto deq_W = ngraph::pattern::wrap_type<ngraph::op::v1::Multiply>({convert_W, weights_scale_W});
+
+    auto R_m = ngraph::pattern::wrap_type<ngraph::op::Constant>();
+    auto convert_R = ngraph::pattern::wrap_type<ngraph::op::Convert>({R_m});
+    auto weights_scale_R = ngraph::pattern::wrap_type<ngraph::op::Constant>();
+    auto deq_R = ngraph::pattern::wrap_type<ngraph::op::v1::Multiply>({convert_R, weights_scale_R});
+
+    const auto B_m = ngraph::pattern::wrap_type<ngraph::opset8::Constant>();
+
+    auto lstm_seq_m  = ngraph::pattern::wrap_type<ngraph::opset8::LSTMSequence>({deq_X, H_in, cell_state_m, sequence_length_m, deq_W, deq_R, B_m});
+    auto gru_seq_m   = ngraph::pattern::wrap_type<ngraph::opset8::GRUSequence> ({deq_X, H_in,               sequence_length_m, deq_W, deq_R, B_m});
```

Comment:
Should we also add the matcher for RNNSequence node?

Reply:
RNNSequence quantization is not supported by oneDNN